source/_includes/custom/search.html

```diff
@@ -18,5 +18,11 @@
       entries : '.entries',         // selector for search entries containing element (contained within results above)
       template: '#search-results-template'  // selector for Mustache.js template
     });
+    $('#search-query').keydown(function(event){
```

Comment:
Minor: but it isn't that obvious to the purpose of this function (besides seeing the notes on the JIRA). Would help by just having a single comment of:

> Currently ignoring the "return/enter" keystroke when searching (as this would just result in a 404 for the page lookup).